src/containers/org/DataSetMetadataContainer.js

```diff
@@ -92,30 +89,28 @@ const DataSetMetaDataContainer = ({
 
   const updateOrgDataSetRS :?RequestState = useRequestState([EDM, UPDATE_ORGANIZATION_DATA_SET]);
 
-  const dataSetColumns :List<Map<FQN, List>> = useSelector(selectOrgDataSetColumns(organizationId, dataSetId));
+  const dataSetColumns :Map<UUID, Map> = useSelector(selectOrgDataSetColumns(organizationId, dataSetId));
   const myKeys :Set<List<UUID>> = useSelector(selectMyKeys());
   const isDataSetOwner :boolean = myKeys.has(List([dataSetId]));
 
   useEffect(() => {
-    // NOTE: the column is ol.column
     const data :List = dataSetColumns
-      .sortBy((column :Map<FQN, List>) => getPropertyValue(column, [FQNS.OL_INDEX, 0]))
-      .map((column :Map<FQN, List>) => ({
-        dataType: getPropertyValue(column, [FQNS.OL_DATA_TYPE, 0]),
-        description: getPropertyValue(column, [FQNS.OL_DESCRIPTION, 0]),
-        id: getEntityKeyId(column),
-        title: getPropertyValue(column, [FQNS.OL_TITLE, 0]),
+      .valueSeq()
+      .map((column :Map) => ({
+        dataType: column.get('dataType'),
+        description: column.getIn(['metadata', 'description']),
+        id: column.get('id'),
+        title: column.getIn(['metadata', 'title']),
```

Comment:
same note as above, should these strings be constants?

Reply:
these strings are throughout the PR, so I'm not gonna continue to leave comments, but just curious if you think this is an appropriate situation for pulling these out into constants.